apps/frontend/src/pages/project/libs/components/project-menu/project-menu.tsx

```diff
@@ -0,0 +1,57 @@
+import { Menu, MenuItem, NavLink } from "~/libs/components/components.js";
+import { AppRoute, PermissionKey } from "~/libs/enums/enums.js";
+import { checkHasPermission } from "~/libs/helpers/helpers.js";
+import { useAppSelector, useCallback, usePopover } from "~/libs/hooks/hooks.js";
+
+type Properties = {
+	onEdit: () => void;
+	onManageAccess?: () => void;
+};
+
+const ProjectMenu = ({
+	onEdit,
+	onManageAccess,
+}: Properties): JSX.Element | null => {
```

Comment:
Why is it optional?

Reply:
It's not merged yet)
It's the project access management page, not the main access

Reply:
I still adjusted it, it doesn't really matter since it will still be changed after the project access management page implementation.